stamp.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
+
+set -e # Exit on errors
+# set -x # Shell debugging
+
+# First, check that we're on the main branch.
+BRANCH=$(git rev-parse --abbrev-rev HEAD)
+if [ "$BRANCH" != "main" ] ; then 
+    echo "Not on main, exiting!" 
+    # exit 1
+fi 
+
+# And check that the main branch is clean
+STATUS=$(git status --porcelain)
+if [ -n "${STATUS}" ] ; then
+    echo "Repository is not clean, exiting!" 
+    # exit 1
+fi 
+
+# And check that the REMOTE points to the -tools repo.
+REMOTE="${REMOTE:origin}"
+if [ "$(git remote get-url $REMOTE 2>/dev/null)" != "git@github.com:awsdocs/aws-doc-sdk-examples-tools.git" ] ; then
```

Comment:
This fails for me because when I run this manually I get
```
$ git remote get-url origin
https://github.com/awsdocs/aws-doc-sdk-examples-tools.git
```

Reply:
Done.

Reply:
This still doesn't work. It ends with `REMOTE=''` for me.